RecervacionVuelos/Gemfile

```diff
@@ -0,0 +1,7 @@
+source 'https://rubygems.org'
```

Comment:
Why is there a Ruby file??

Reply:
It is a default file for React Native when using Mac to emulate with Ios, it contains only cocoapods version 😸